### ISSUE TITLE
Review the new Lift Access, Usage and Maintenance Wiki Page

### ISSUE THREAD

**Issue by chris-does** [DoESLiverpool/somebody-should#1320]
### Lift Access, Usage and Maintenance

Following a recent sounding of the lift alarm (which was not related to any risk of harm or injury), a lift engineer attended site to reset the alarm.

I have now documented a lift alarm reset process that does not require an engineer to attend site. 
As I was writing it up, I also made some notes on instructions to use the lift.

### Action taken

I have created a new [Lift Access, Usage and Maintenance](https://github.com/DoESLiverpool/somebody-should/wiki/Lift-Access,-Usage-and-Maintenance) page on the wiki. Please accept my apologies - and point me to it - if this already exists somewhere else, I was not able to find it when I looked.

I have also updated the [Location](https://github.com/DoESLiverpool/somebody-should/wiki/Location#where-we-are) wiki page to reference the new [lift](https://github.com/DoESLiverpool/somebody-should/wiki/Lift-Access,-Usage-and-Maintenance) page on the wiki. I am not sure if this new [lift](https://github.com/DoESLiverpool/somebody-should/wiki/Lift-Access,-Usage-and-Maintenance) page should be linked to from anywhere else.

### Action required to close this issue

I would be most grateful if someone (or a few people) could review the content of the new [lift](https://github.com/DoESLiverpool/somebody-should/wiki/Lift-Access,-Usage-and-Maintenance) page.

By all means make edits/additions/deletions as you see fit, or alternatively specify in the comments below any changes you would like me to make to the new [lift](https://github.com/DoESLiverpool/somebody-should/wiki/Lift-Access,-Usage-and-Maintenance) page.

### Input sought

One item, in particular, I would like some guidance on is the "Reporting to The Tapestry" (Number 20). I do believe this is necessary, but am not sure of any useful contact details to include in that item. Or, indeed, whether the creation of the described somebody-should issue would be sufficient. Perhaps a tag for the issue would be enough? Maybe Building or Health & Safety? Or is this a suitable reason to create a `Building: Landlord` or equivalent tag, for somebody-should issues which must receive confirmation from the Landlord before being closed . . .
